lib/srv/reexec.go

```diff
@@ -138,6 +165,18 @@ func RunCommand() (io.Writer, int, error) {
 			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("pty and tty not found")
 		}
 		errorWriter = tty
+		ttyName, err = os.Readlink(tty.Name())
+		if err != nil {
+			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("failed to resolve tty soft link: %v", err)
+		}
+		soft, err := createUaccSession(ttyName, &c)
+		if err != nil {
+			if soft {
+				log.WithError(err).Warnf("failed to register closed interactive session for tty %s in the system account database with error %s", ttyName, trace.Wrap(err).Error())
```

Comment:
Capitalization.

Does this mean if I run Teleport in a shell every time I login I will see a `Warnf`?

Reply:
If it lacks utmp permissions, yes. Is this not desired behavior?

Reply:
@a-palchikov said errors should be lowercase to be composable and that's how they are in other places of the codebase.

Reply:
Yes, I meant error values when they're returned from a function. The log messages are traditionally written as separate sentences:
```go
return trace.BadParameter("invalid argument")
```
vs
```go
log.WithError(err).Warn("Failed to close the file.")
```

Reply:
Ah, adjusted, thanks for clarification.